README.md

```diff
@@ -57,6 +57,16 @@ You also need to add the required API permissions to have '`Azure DevOps user_im
 
 ![aad api permissions](https://imgur.com/aVd51d0.png)
 
+### register-azure-devops-npm-auth.ps1
+
+To automate the creation of the above AD app registration in *your* AD tenant:
+
+1. Ensure you have installed [azure-cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+2. Open a powershell core prompt (minimum version 6.1)
```

Comment:
does this need to be run as an administrator?

Reply:
yeah the script mentions that you need Global administrator (line 17). This is as per the docs:

<img width="802" alt="image" src="https://user-images.githubusercontent.com/886590/155394616-2636b364-1d8d-4230-8889-3dc0c1708f91.png">

That, said I think a lesser role might be sufficient (see below), but I hedged my bets to go with the above.

<img width="1314" alt="image" src="https://user-images.githubusercontent.com/886590/155395055-5b102ca8-11de-4b66-8697-ba3a75e898e9.png">

Reply:
I meant the PowerShell prompt probably needs to be "Run as administrator..." right?

Reply:
that a windows thing - I'm on a mac :-)

I'm not sure but I wouldn't expect a windows admin credential (or lack of) would affect the azure login session you acquire via `az login`... but I could well be wrong!

Reply:
Yo @yohanb, any more information you need from me?

Reply:
@christianacca nope!  All good!  Will merge and publish asap :)

Reply:
actually no need to publish the package; PR is merged back!  Thanks for your contribution :+1: